mystery_word.py

```diff
@@ -0,0 +1,103 @@
+import random
+import re
+
+with open("/usr/share/dict/words") as w:
+    word_list = w.readlines()
+
+guessed = []
+revealed = []
+
```

Comment:
This stuff should be at the bottom in a `if __name__ == "__main__"` block, along with the rest of the code in the bottom of this file.